### PR TITLE
translations for 

### PR DESCRIPTION
**Pull request by CringeArab**
Translations for [YimMenu#3380](https://github.com/YimMenu/YimMenu/pull/3380)

### CODE REVIEW

**Review by gir489returns — CHANGES_REQUESTED** [review]
BACKEND_LOOPED_WEAPONS_AIM_ONLY_AT_PLAYER doesn’t match the form used in AT ENEMY key (different падеж, idk how to translate it)
DESTROY_PERSONAL_VEHICLE doesn’t have any mention that vehicle is personal. 
PED_TYPE_PROSTITUTE has an interesting translation, it matches just fine, but if to be pedantic it should’ve been «Проститутки», it might not fit again, idk. Or you worked in some translation studio where you are discouraged from using direct swears even if original had them lmao.
TSE_FREEZE_MISMATCH is “install mismatch of script event”

File: en_US.json [hunk 1]
```diff
@@ -1471,6 +1488,8 @@
   "TRUST_FRIENDS_DESC": "Friends won't be flagged as modders or taken actions by reactions",
   "TRUST_SESSION": "Trust this session",
   "TRUST_SESSION_DESC": "Players in this session won't be flagged as modders or taken actions by reactions",
+  "TSE_FREEZE": "TSE Freeze",
+  "TSE_FREEZE_MISMATCH": "Install Mismatch of Script Event",
```
Comment (gir489returns) [inline en_US.json:1492]: This entry makes no sense, what are you installing?

**Review by gir489returns — CHANGES_REQUESTED** [review]

File: en_US.json (hunk 1 above)
Comment (gir489returns) [reply en_US.json]: Fix it please.

File: ru_RU.json [hunk 2]
```diff
@@ -1489,7 +1489,7 @@
   "TRUST_SESSION": "Доверять этой сессии",
   "TRUST_SESSION_DESC": "Игроки в данной сессии не будут отмечены, как моддеры, и против них не будет реакционных действий",
   "TSE_FREEZE": "Зависание игры",
-  "TSE_FREEZE_MISMATCH": "Установка несоответствия скриптового события",
+  "TSE_SENDER_MISMATCH": "Установка несоответствия скриптового события",
```
Comment (gir489returns) [inline ru_RU.json:1492]: Make this TSE Эмиттер несоответствие. Double check it, my Russian is trash.

**Review by Yimura — APPROVED** [review]

**Pull request merged**